src/TransformOperationExecutor.ts

```diff
@@ -88,6 +88,12 @@ export class TransformOperationExecutor {
         } else if (targetType === Boolean && !isMap) {
             if (value === null || value === undefined)
                 return value;
+            if (value === 'true' || value === '1' || value === 'yes') {
+                return true;
+            }
+            if (value === 'false' || value === '0' || value ==='no') {
+                return false;
+            }
```

Comment:
`yes/no` is a bit too much. I recommend to remove them.

Please also remove the `{}`s, the rest of the code does not have them for single line if statements.

We also need some tests.

Reply:
You may check the tests from [this PR](https://github.com/typestack/class-transformer/pull/363/files) if you need ideas.